recipes/libnl/all/conanfile.py

```diff
@@ -17,61 +20,46 @@ class LibNlConan(ConanFile):
     default_options = {"fPIC": True, "shared": False}
     build_requires = ( "flex/2.6.4", "bison/3.7.6" )
```

Comment:
This line was not changed in this PR, but the idiomatic way seems to be using [build_requirements](https://docs.conan.io/2.0/reference/conanfile/methods/build_requirements.html) instead

Reply:
There's a chance this can be removed - in Conan Center we assume that Linux/macOS have a baseline set of tools installed at the system level (including these), so it's worth trying a build removing these. I'll make these changes on top of some small corrections :)